R/dashbioDashNgl.R

```diff
@@ -0,0 +1,18 @@
+# AUTO GENERATED FILE - DO NOT EDIT
+
+DashNglDashNgl <- function(id=NULL, viewportStyle=NULL, stageParameters=NULL, data=NULL) {
```

Comment:
What is this file? Obsolete?

Reply:
Yes, thanks for pointing it out. I've rebuilt the R package and deleted this file in 9f05326.